programs/rodeos/ship_client.hpp

```diff
@@ -208,11 +212,11 @@ struct connection : connection_base {
       } catch (...) { elog("exception while closing"); }
    }
 
-   void close(bool retry) {
-      ilog("closing state-history socket");
+   void close(bool retry, bool quitting = false) {
```

Comment:
Would prefer for `quitting` not to be defaulted.

Reply:
Made it not defaulted.